server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/ProjectActivityPage.java

```diff
@@ -46,18 +43,18 @@ public ElementsCollection getAnalyses() {
   }
 
   public ProjectAnalysisItem getLastAnalysis() {
-    return new ProjectAnalysisItem($(".project-activity-analysis"));
+    return new ProjectAnalysisItem(Selenide.$(".project-activity-analysis"));
   }
 
   public ProjectAnalysisItem getFirstAnalysis() {
-    return new ProjectAnalysisItem($$(".project-activity-analysis").last());
+    return new ProjectAnalysisItem(Selenide.$$(".project-activity-analysis").last());
   }
 
   public ProjectActivityPage assertFirstAnalysisOfTheDayHasText(String day, String text) {
-    $("#project-activity")
+    Selenide.$("#project-activity")
       .find(".project-activity-day[data-day=\"" + day + "\"]")
       .find(".project-activity-analysis")
-      .should(hasText(text));
+      .should(Condition.hasText(text));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "hasText"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)